functions/bigtable/main_test.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datetime
+import os
+import uuid
+import pytest
```

Comment:
`pytest` is a 3rd party library. Can you do the following

```python

import datetime
import os
import uuid

from google.cloud import bigtable
from requests import Request
import pytest

import main
```

?

Reply:
I still see `pytest` is in standard library block.

Reply:
Sorry, I fixed this, but was getting this when pushing which I guess prevented it from going in

```
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing...
```